deploy/kubernetes/che-devfile-registry/templates/deployment.yaml

```diff
@@ -10,15 +10,21 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  labels:
-    app: che-devfile-registry
   name: che-devfile-registry
+  labels:
+    app: che
+    component: devfile-registry
+    app.kubernetes.io/name: che
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 spec:
   replicas: 1
   revisionHistoryLimit: 2
   selector:
     matchLabels:
-      app: che-devfile-registry
+      app: che
+      component: devfile-registry
```

Comment:
Should not it be replaced with `app.kubernetes.io/component` from https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ ?

Reply:
We keep those labels in che-operator since `selector` field is immutable. 
So I decided to keep the same labels here as well.
```
app: che
component: devfile-registry
```